Recompiler CPU.c

```diff
@@ -2904,9 +2904,6 @@ void __cdecl StartRecompilerCPU (void ) {
 			DWORD Value;
 
 			for (;;) {
-				if (CPU_Action.CloseCPU) {
-					DoSomething();
```

Comment:
This function name is 🤮! It's a generic catchall for handling periodic things like timers and checking if it's time to stop the CPU thread.

I added this here while doing initial research in #45, but that PR was merged a bit early. So, I'm cleaning it up now!